src/OpenTelemetry/Trace/Tracer.cs

```diff
@@ -85,22 +85,12 @@ public IDisposable WithSpan(ISpan span, bool endSpanOnDispose)
         /// <inheritdoc/>
         public ISpan StartRootSpan(string operationName, SpanKind kind, SpanCreationOptions options)
         {
-            if (operationName == null)
```

Comment:
Emit logs when starting new root span with invalid params.

Reply:
done in span constructor